pkg/project/project.go

```diff
@@ -1,12 +1,12 @@
 package project
 
 var (
-	bundleVersion = "0.23.18"
+	bundleVersion = "0.23.19"
```

Comment:
I will not mess up with bundleVersion again.

Reply:
Good catch, thanks!